src/main/java/core/basesyntax/service/impl/BetMapperImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.Bet;
+import core.basesyntax.service.Mapper;
+
+public class BetMapperImpl implements Mapper<Bet> {
+    @Override
+    public Bet parse(String[] data) {
+        if (data == null) {
+            throw new RuntimeException("Expected to receive data, but got null");
+        }
+        try {
+            int value = Integer.parseInt(data[0]);
+            double risk = Double.parseDouble(data[1]);
+            return new Bet(value, risk);
+        } catch (NumberFormatException e) {
+            throw new NumberFormatException("Incorrect data");
```

Comment:
this doesn't make sense
let's either don't catch this, or print out a message inside a catch block if we want our program to continue